src/components/Preview.jsx

```diff
@@ -13,7 +13,7 @@ class Preview extends React.Component {
     bindAll(this, '_handlePopOutClick');
   }
 
-  _generateDocument() {
+  _generateDocument(isPopout = false) {
```

Comment:
Any reason to take this as a parameter, since it’ll always be `true` in the context of the `Preview` component?